script.js

```diff
@@ -22,8 +21,22 @@ const interestingTypes = [
   'locality',
 ];
 
+const defaultLangValue = 'English (US)';
+const languageMap = {
```

Comment:
Would you mind adding a few more options from the large wikipedias
https://en.wikipedia.org/wiki/List_of_Wikipedias#Detailed_list

Reply:
Done, I adde the major ones. Welcome sentence is just a google translate hopefully they all make sense ^^